docs/tutorials/getting_started/tutorial_overview.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Getting started with Great Expectations
+---
+
+Welcome to the Great Expectations getting started tutorial! This tutorial will help you set up your first local deployment of Great Expectations.  This deployment will contain a small Expectation Suite that we will use to validate some sample data. We'll also introduce important concepts, with links to detailed material you can dig into later.
+
+:::tip
+The steps described in this tutorial assume you are installing Great Expectations version 0.13.8 or above.
+
+For a tutorial for older versions of Great Expectations, please see older versions of this documentation, which can be found [here](https://docs.greatexpectations.io/en/latest/guides/tutorials.html).
+:::
+
+### This tutorial will walk you through the following steps
+
+<table class="borderless">
+<tr>
+    <td>
+        <img
+          src={require('../../images/universal_map/Gear-active.png').default}
+          alt="Setup"
+        />
+    </td>
+    <td>
+        <h4>Setup</h4>
+        <p>First, we will make sure you have Great Expectations installed and show you how to initialize a <b>Data Context.</b></p>
+    </td>
+</tr>
+<tr>
+    <td>
+        <img
+          src={require('../../images/universal_map/Outlet-active.png').default}
+          alt="Connect to Data"
+        />
+    </td>
+    <td>
+        <h4>Connect to Data</h4>
+        <p>Then you will learn how to configure a <b>Datasource</b> to connect to your data.</p>
+    </td>
+</tr>
+<tr>
+    <td>
+        <img
+          src={require('../../images/universal_map/Flask-active.png').default}
+          alt="Create Expectations"
+        />
+    </td>
+    <td>
+        <h4>Create Expectations</h4>
+        <p>You will then create your first <b>Expectation Suite</b> using the built-in automated <b>Profiler</b>. We'll also give you a tour of <b>Data Docs</b> to view your <b>Validation Results</b>.</p>
```

Comment:
I think we need to move the reference to viewing Validation Results into the Validate data section, both here and when it appears; is the current presentation of data docs in the "create expectations" section focused on the prescriptive view?

Reply:
It is.  After the Profiler runs, Data Docs are brought up where the user can see what Expectations were created by the Profiler.  I agree, this should state that they will be using Data Docs to view the Expectations created by the Profiler, not any Validation Results.

Reply:
Update has been pushed.